===================================================================
RCS file: /cvs/src/src/binutils/cxxfilt.c,v
retrieving revision 1.14
retrieving revision 1.15
diff -u -r1.14 -r1.15
--- src/binutils/cxxfilt.c	2007/07/05 16:54:45	1.14
+++ src/binutils/cxxfilt.c	2007/11/26 01:53:59	1.15
@@ -63,12 +63,12 @@
   result = cplus_demangle (mangled_name + skip_first, flags);
 
   if (result == NULL)
-    printf (mangled_name);
+    printf ("%s", mangled_name);
   else
     {
       if (mangled_name[0] == '.')
 	putchar ('.');
-      printf (result);
+      printf ("%s", result);
       free (result);
     }
 }
